xrpl-std/src/core/field_codes.rs

```diff
@@ -0,0 +1,61 @@
+pub const SF_ACCOUNT: i32 = 524289; // 0x80001
+pub const SF_TRANSACTION_TYPE: i32 = 65538; // 0x10002
+pub const SF_FEE: i32 = 393224; // 0x60008
+pub const SF_SEQUENCE: i32 = 131076;
+pub const SF_ACCOUNT_TXN_ID: i32 = 327689;
+pub const SF_FLAGS: i32 = 131074;
+pub const SF_LAST_LEDGER_SEQUENCE: i32 = 131099;
+pub const SF_MEMOS: i32 = 983049;
```

Comment:
These codes are required by `xrpl-std` so need to live in here. I think this means we can get rid of `sfield.rs` inside of the `wasm-host` project.

Reply:
After speaking with @pwang200 it's helpful to have this in both places, so need to combine this with `sfield.rs` in std lib.